modules/platform/nuxeo-fsexporter/src/main/java/org/nuxeo/io/fsexporter/DefaultExporterPlugin.java

```diff
@@ -138,4 +142,16 @@ protected String getFileName(Blob blob, DocumentModel docfrom, File folder, int
 
         return prefix + blob.getFilename();
     }
+
+    @Override
+    public DocumentModelList getCollectedDocuments(CoreSession session, DocumentModel collection) {
+        Collection collectionAdapter = collection.getAdapter(Collection.class);
+        List<String> collectedDocumentIds = collectionAdapter.getCollectedDocumentIds();
+        List<DocumentModel> collectedDocumentsList = new ArrayList<DocumentModel>();
+        for (int i = 0; i < collectedDocumentIds.size(); i++) {
+            collectedDocumentsList.add(session.getDocument(new IdRef(collectedDocumentIds.get(i))));
+        }
+        DocumentModelList collected = new DocumentModelListImpl(collectedDocumentsList);
+        return collected;
```

Comment:
You could use the functional API there.
```suggestion
        return collectionAdapter.getCollectedDocumentIds()
                                .stream()
                                .map(IdRef::new)
                                .map(session::getDocument)
                                .collect(Collectors.collectingAndThen(Collectors.toList(), DocumentModelListImpl::new));
```
Note: the two methods from _Collectors_ could be imported statically.